py3status/modules/hamster.py

```diff
@@ -27,19 +25,15 @@ class Py3status:
     format = '{current}'
 
     def hamster(self):
-        cur_task = check_output(shlex.split('hamster current'))
-        cur_task = cur_task.decode('ascii', 'ignore').strip()
+        cur_task = self.py3.command_output('hamster current')
         if cur_task != 'No activity':
             cur_task = cur_task.split()
             time_elapsed = cur_task[-1]
             cur_task = cur_task[2:-1]
             cur_task = "%s (%s)" % (" ".join(cur_task), time_elapsed)
```

Comment:
![2017-01-29__09 57 37](https://cloud.githubusercontent.com/assets/852504/22405508/784c0234-e609-11e6-871d-54f7db7c38a8.png)


If I add `.decode('ascii', 'ignore').strip()`, I get this exception below so it's not needed anymore. I'll keep `strip()` just in case.
```
2017-01-29 09:47:17 WARNING Instance `hamster`, user method `hamster` failed (AttributeError) hamster.py line 28.
2017-01-29 09:47:17 INFO Traceback
AttributeError: 'str' object has no attribute 'decode'
  File "/home/chris/src/py3status/py3status/module.py", line 584, in run
    response = method()
  File "/home/chris/src/py3status/py3status/modules/hamster.py", line 28, in hamster
    cur_task = self.py3.command_output('hamster current').decode('ascii', 'ignore').strip()
```